src/core/checkedint.d

```diff
@@ -492,8 +494,8 @@ unittest
 pragma(inline, true)
 ulong mulu(ulong x, ulong y, ref bool overflow)
 {
-    ulong r = x * y;
-    if (x && (r / x) != y)
+    const r = x * y;
+    if (x && r / x != y)
```

Comment:
This is ``a && b / c != d``, parenthesis really does help make it clear at a glance just exactly what it is doing.